src/payment/__snapshots__/PaymentPage.test.jsx.snap

```diff
@@ -430,16 +412,17 @@ exports[`<PaymentPage /> Renders correctly in various states should render all c
                 className="h6 d-block"
                 htmlFor="couponField"
               >
-                Add coupon code (optional)
+                <span>
```

Comment:
How much do we care about these extra span tags?  If you use the version of FormattedMessage that takes a function as its child, you should be able to coax it not to use any tags at all, I think.

Reply:
I think they do no harm, I can checkout the function version, but if it's syntactically messy I'd prefer we just accept these spans.